### DISCUSSION TITLE
View Cell output on the side instead of under

### DISCUSSION THREAD

**Discussion by vedal** [microsoft/vscode-jupyter#9404] [New Ideas]
I'm curious if its possible to achieve a similar behavior to jupyter lab here, where cell output is shown on the side rather than under. 
Would be great for avoiding scrolling up and down all the time for longer cells.
Example jupyterlab-video: https://www.youtube.com/watch?v=Asa_ML45HP8
Also requested for jupyter notebook here: https://github.com/jupyter/notebook/issues/2897

**Comment by amunger** [comment]
There was a good amount of discussion about this kind of thing in this [VS Code issue](https://github.com/microsoft/vscode/issues/143244)

**Reply by vedal** [reply]
Thanks alot!

**Reply by IanMatthewHuff** [reply]
@vedal If you are interested, upvotes on that issue are basically votes for us implementing something like that. So make sure to give it a thumbs up if that would suit your usage.

**Reply by vedal** [reply]
@IanMatthewHuff I upvoted the two top comments, shppose thats what you mean. Thanks for the tip! :)
